docs/xamarin-forms/creating-your-first-prism-app.md

```diff
@@ -74,7 +74,7 @@ The `App.xaml.cs` file contains the logic required to configure a Prism applicat
          * This imposes a limitation in which the App class must have a default constructor.
          * App(IPlatformInitializer initializer = null) cannot be handled by the Activator.
          */
-        public App() : this(null) { }
+        public App() : base(null) { }
```

Comment:
The original approach is correct. We want to call `this(null)` to ensure that any cod placed in the overloaded ctor will be executed.  If you use `base(null)` then any code in the overloaded ctor will not be invoked and could cause bugs in your code.

Reply:
humm. I need to review some apps that I made.. 😅